src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.tsx

```diff
@@ -36,47 +47,28 @@ const formatDateRange = (startDate?: DateTime, endDate?: DateTime) => {
 };
 
 export const AccountTransactions: React.FC = () => {
-  const { query } = useRouter();
   const { t } = useTranslation();
   const locale = useLocale();
-  const {
-    accountListId,
-    financialAccountId,
-    activeFilters,
-    setActiveFilters,
-    hasActiveFilters,
-    searchTerm,
-    setPanelOpen,
-  } = useContext(FinancialAccountContext) as FinancialAccountType;
+  const { accountListId, financialAccountId, setPanelOpen } = useContext(
+    FinancialAccountContext,
+  ) as FinancialAccountType;
+
+  const { activeFilters, searchTerm } =
+    useUrlFilters<FinancialAccountsFilters>();
+
   const { appName } = useGetAppSettings();
 
   useEffect(() => {
     // On loading the transactions page, open the filters panel and reset the active filters.
     // We need to reset the active filters to ensure the date range is set to the date range in the URL if it exists.
```

Comment:
Sorry, I know I asked you to add this back. But after my changes, the comment isn't completely accurate anymore.

```suggestion
    // On loading the transactions page, open the filters panel
```